src/sources/vector.rs

```diff
@@ -25,12 +25,13 @@ fn default_shutdown_timeout_secs() -> u64 {
     30
 }
 
+#[cfg(test)]
```

Comment:
This seems like it might have slipped in, is this intentional?

Reply:
Very intentional. The tests for both the sink and source are in this file, so this is required to test TLS support.

Reply:
Ok!